src/lib/builders/context-menu/create.ts

```diff
@@ -111,12 +111,14 @@ export function createContextMenu(props?: CreateContextMenuProps) {
 		const target = e.target;
 		if (!(target instanceof Element)) return;
 
-		if (target.id === ids.trigger && isLeftClick(e)) {
+		const isClickInsideTrigger = target.closest(`#${ids.trigger}`) !== null;
+
+		if (isClickInsideTrigger && isLeftClick(e)) {
```

Comment:
```suggestion
		if (!isClickInsideTrigger || isLeftClick(e)) {
```
And then delete the other if 😄